src/hooks/useGraphQLClient.tsx

```diff
@@ -38,7 +38,7 @@ export const GraphQLClientContextProvider = ({
     injectedGraphQLInstance || defaultGraphQLClientInstance;
 
   if (baseURL) {
-    graphQLInstance.setEndpoint(baseURL || defaultBaseURL);
+    graphQLInstance.setEndpoint(`${baseURL}/v1/graphql` || defaultBaseURL);
```

Comment:
Just seeing this, but it seems like the `|| defaultBaseURL` here could never have been used.  It's just more glaring now

Reply:
Oh yeah, 100%, I see that now.

Reply:
Fixed